### DISCUSSION TITLE
Welcome to ReVancedUpdater Discussions!

### DISCUSSION THREAD

**Discussion by LeddaZ** [LeddaZ/ReVancedUpdater#35] [Announcements]
This is a place for suggestions, since the issues sections is, as the name says, to report issues or bugs.

**Comment by B0nJamTam** [comment]
Coucou, j’espere que tu vas bien 😁. Tout d'abord merci beaucoup et de nous permettre de profiter de ton/votre travail. Petite suggestion totalement utile/inutile mais remplacer l’image app par celle de YT off.
Petite question pourquoi MicroG RE ne s'affiche pas dans nos applications dans notre launcher mais bien dans la section paramètre de nos android ?

**Comment by B0nJamTam** [comment]
Petite suggestion en plus : lorsque on utilise cloudflare warp, ils nous demande de confirmer que l'on n’est un pas un robot en se connectant à un compte seulement perso je n’ai pas envie de connecter mon compte donc je dois fermer l’app une fois ou plusieurs fois pour ne plus avoir cette confirmation.

**Comment by accnetdev** [comment]
Bonjour et merci à LeddaZ pour Morphe Updater.
Je mentionne juste un problème qui n'a rien à voir dans l'espoir d'avoir des commentaires et retours d'expériences : mon YouTube Morphe met beaucoup de temps à lire les vidéos ça finit par démarrer mais on dirait qu'il doit precharger alors qu'avec NewPipe c'est quasi immédiat. Vivez vous le même comportement ? Comment diagnostiquer ce problème ?
